.github/workflows/tests.yaml

```diff
@@ -32,7 +32,8 @@ jobs:
       # Install all dependencies at the root
       - name: Install dependencies (pnpm)
         run: pnpm install
-
+      - name: Check formatting
+        run: pnpm format:check
```

Comment:
wouldn't it be best to use the [biome ci](https://biomejs.dev/reference/cli/#biome-ci)?

Reply:
Will do